.travis.yml

```diff
@@ -1,29 +1,35 @@
 language: ruby
+cache: bundler
+
 before_install:
   - gem install -v '1.17.3' bundler
-install: bundle _1.17.3_ install
-bundler_args: --jobs 3 --retry 3 --without benchmark
+bundler_args: --without benchmark
+
 os:
   - linux
   - osx
+
 rvm:
-  - 2.5.0
-  - 2.4.3
-  - 2.3.5
-  - 2.2.9
-  - 2.1.10
-  - 2.0.0
-  - 1.9.3
+  - 2.7
+  - 2.6
+  - 2.5
+  - 2.4
+  - 2.3
+  - 2.2
+  - 2.1
+  - 2.0
+  - 1.9
+
 matrix:
   exclude:
-    - rvm: 2.0.0
+    - rvm: 2.0
       os: osx
-    - rvm: 1.9.3
+    - rvm: 1.9
       os: osx
   include:
-    - rvm: 2.0.0
+    - rvm: 2.0
       os: osx
       osx_image: xcode7.3
```

Comment:
These images are kind of old, should we add newer ones or maybe even replace these?

https://docs.travis-ci.com/user/reference/osx#macos-version

Reply:
Ah, my bad, this is for testing with old Rubies, right? I assume these are not available on newer macOS versions.

https://github.com/tonytonyjan/jaro_winkler/commit/868609afb5530125dacecd221b9dd50c7f0c84b9